easybuild/easyconfigs/b/Bazel/Bazel-3.7.2-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,34 @@
+name = 'Bazel'
+version = '3.7.2'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/bazelbuild/%(namelower)s/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+patches = [
+    '%(name)s-3.4.1-fix-grpc-protoc.patch',
+    '%(name)s-%(version_major_minor)s.1_fix-protobuf-env.patch',
+]
+checksums = [
+    'de255bb42163a915312df9f4b86e5b874b46d9e8d4b72604b5123c3a845ed9b1',  # %(namelower)s-%(version)s-dist.zip
+    'f87ad8ad6922fd9c974381ea22b7b0e6502ccad5e532145f179b80d5599e24ac',  # %(name)s-3.4.1-fix-grpc-protoc.patch
+    '8706ecc99b658e0a96c38dc2c23e44da35059b85f308602aac76a6d6680376e7',  # %(name)s-%(version_major_minor)s.1_fix-protobuf-env.patch
```

Comment:
@casparvl could you put the comment underneath the checksum to appease the test suite?